src/types/objectsPosition.ts

```diff
@@ -0,0 +1,87 @@
+export const OBJECTS_POSITION_OPTIONS = [
+  "no",
+  "north",
```

Comment:
There are no "full" names for gravity types

Reply:
Oh, hmm. I misread the documentation. From the first glance it looks like it accepts both like for names. Fixed